src/routes/comment.tsx

```diff
@@ -144,17 +152,19 @@ async function renderCommentOGImage(c: any, commentId: string) {
                   overflow: "hidden",
                   textOverflow: "ellipsis",
                   whiteSpace: "nowrap",
+                  // gap is needed to fix text ellipsis
+                  gap: "8px",
                 }}
               >
-                By{" "}
-                <UserName
-                  name={artistName}
-                  isVerified={isArtistVerified}
-                  tier={artistTier}
-                  backgroundColor={dominantColor}
-                  style={{ fontSize: "40px", fontWeight: "500", lineHeight: "48px" }}
-                />
+                By {artistName}
               </p>
+              <UserBadge
+                isVerified={isArtistVerified}
```

Comment:
Why no UserName?

Reply:
it messed up the truncation in this case